src/platform/unix/mod.rs

```diff
@@ -79,6 +81,21 @@ pub fn channel() -> Result<(OsIpcSender, OsIpcReceiver),UnixError> {
     }
 }
 
+#[derive(Clone, Copy)]
+struct FDHandle {
+    pub id: u64,
+    pub descriptor: c_int
+}
```

Comment:
After thinking about this, I think it is more informative if it is named something like `PollEntry` over `FdEntry`.